torch/testing/_internal/common_utils.py

```diff
@@ -816,6 +816,25 @@ def settings(*args, **kwargs):
 except ImportError:
     print('Fail to import hypothesis in common_utils, tests are not derandomized')
 
+
+slow_tests_dict: Optional[Dict[str, float]] = None
+def check_slow_test_from_stats(test):
+    global slow_tests_dict
+    if slow_tests_dict is None:
+        url = 'https://raw.githubusercontent.com/janeyx99/test-infra/main/stats/.pytorch-slow-tests'
+        contents = urlopen(url, timeout=1).read().decode('utf-8')
+        slow_tests_dict = json.loads(contents)
+    slow_tests_dict = cast(Dict[str, float], slow_tests_dict)
```

Comment:
This line seems surprising since the previous line already assigns to `slow_tests_dict`; is the difference that the previous line assigns to a local value while this line assigns to the `global slow_tests_dict` value?

Reply:
Yes

Reply:
I investigated this further and both refer to the global variable, so I am going to remove the cast as it's confusing.